src/concern/test.rs

```diff
@@ -363,34 +351,24 @@ async fn command_contains_write_concern_update_many() {
     coll.insert_many(&[doc! { "foo": "bar" }, doc! { "foo": "bar" }], None)
         .await
         .unwrap();
-    coll.update_many(
-        doc! { "foo": "bar" },
-        doc! { "$set": { "foo": "baz" } },
-        UpdateOptions::builder()
-            .write_concern(
-                WriteConcern::builder()
-                    .w(Acknowledgment::Nodes(1))
-                    .journal(true)
-                    .build(),
-            )
-            .build(),
-    )
-    .await
-    .unwrap();
-    coll.update_many(
-        doc! { "foo": "baz" },
-        doc! { "$set": { "foo": "quux" } },
-        UpdateOptions::builder()
-            .write_concern(
-                WriteConcern::builder()
-                    .w(Acknowledgment::Nodes(1))
-                    .journal(false)
-                    .build(),
-            )
-            .build(),
-    )
-    .await
-    .unwrap();
+    coll.update_many(doc! { "foo": "bar" }, doc! { "$set": { "foo": "baz" } })
+        .write_concern(
```

Comment:
Sidebar: I wonder if it would be worth also having convenience methods for those options where the value itself has a builder.  It could pre-construct a builder and accept a closure that can update it, e.g.:
```
coll.update_many(doc! { "foo": "bar" }, doc! { "$set": { "foo": "baz" } })
    .build_write_concern(|wc| wc.w(Acknowledgement::Nodes(1)).journal(true))
    .await
```
Better to keep it out of this update just to streamline the release, but if you think that'd be a useful addition I'll file a ticket for it.

Reply:
Yup, agreed.  Filed RUST-1865 for this.